### PR TITLE
Middle Name Cafe

### PR DESCRIPTION
**Pull request by matthiasrossini**
New cafe + images

### CODE REVIEW

File: data/taipei/middle-name-cafe.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+website: https://www.middlenamecoffee.com/
+images:
+  - coffee.jpg
+  - interior1.jpg
+  - interior2.jpg
+---
+
+Bright, modern cafe near Zhongshan with both counter, bar, and table seats available. Very fast wifi (160 Mb/s) and plenty of power outlets near the seats. The coldbrew coffee is quite strong as well. Came on a Tuesday and although quite full at times, there was always seating available. 
+
+Submitted by: instagram.com/matthiasrossini
```
Comment (pqvst) [inline data/taipei/middle-name-cafe.md:47]: Author should be added in the YAML above:
```
---
author: https://www.instagram.com/matthiasrossini/
...
---
```

File: data/taipei/middle-name-cafe.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+updated: 2021-01-26
```
Comment (pqvst) [inline data/taipei/middle-name-cafe.md:2]: If you're adding a new place for the first time, use `added: <date>` instead of `updated: <date>`

**Review by pqvst — CHANGES_REQUESTED** [review]
Looks good 👍  I added two small comments!

**Comment by matthiasrossini** [comment]
Just added the changes, let me know if that works!

**Pull request merged**